docs/rackspace-infrastructure-deployment.md

```diff
@@ -0,0 +1,83 @@
+## Rackspace Infrastructure Deployment 
+
+### Ironic for bare-metal provisioning 
+
+Rackspace TEchnology uses OpenStack bare metal provisioning, a.k.a **Ironic**, which provides bare metal machines instead of virtual machines, forked from the Nova baremetal driver. It is best thought of as a bare metal hypervisor API and a set of plugins which interact with the bare metal hypervisors. By default, it will use PXE and IPMI in order to provision and turn on/off machines, but Ironic also supports vendor-specific plugins which may implement additional functionality. 
+
+After switch and firewall configuration, deployment nodes are created with in the environment which host the required Ironic services:
+
+- PXE
+- DHCP
+- NBP
+- TFTP
+- IPMI
+
+### Ironic Diagram
+
+![conceptual_architecture](./assets/images/conceptual_architecture.png)
+
+
+
+#### Benefits of Ironic
+
+​	With a standard API and lightweight footprint, Ironic can serve as a driver for a variety of bare metal infrastructure. Ironic allows users to manage bare metal infrastructure like they would virtual  machines and provides ideal infrastructure to run container  orchestration frameworks like Kubernetes to optimize performance.  
+
+
+
+#### Integration Methods and Use-Case
+
+​	Ironic is being used today to deploy the infrastructure for Rackspace's Openstack Flex Cloud in an internal product known as Undercloud. Undercloud is agnostic to internal Rackspace tools making it flexible to use in our datacenter and also for automated delployments in non-Rackspace datacenters.
+
+​	For example, a customer in their own datacenter can stand up the reference architecture hardware for Rackspace Flex and, granting ingress access to DRAC or iLO, allow Rackspace engineers to create the Ironic environment and flavors required for the compute hardware.  The servers are identified their first PXE boot and the proper server flavor and image are applied.  When a server goes offline to due hardware or drive failures, it can be re-provisioned after repair back to its original operationg system and added back to the Openstack environment through Flex automation.
```

Comment:
Please reference "Rackspace Flex" as OpenStack public cloud.
Also typo "operationg" and Openstack should be referred to as OpenStack